packages/dx-react-grid/src/plugins/table-group-row.tsx

```diff
@@ -106,12 +116,14 @@ class TableGroupRowBase extends React.PureComponent<TableGroupRowProps> {
                           {...params}
                           contentComponent={Content}
                           iconComponent={Icon}
+                          containerComponent={Container}
                           row={params.tableRow.row}
                           column={params.tableColumn.column!}
                           expanded={expandedGroups.indexOf(params.tableRow.row.compoundKey) !== -1}
                           onToggle={() => toggleGroupExpanded(
                             { groupKey: params.tableRow.row.compoundKey },
                           )}
+                          left={contentLeft}
```

Comment:
Can we rename this property and others similar? If we are going to create an RTL support for Grid, this property should be renamed.

We can use `indent` name instead.